core/shared/src/main/scala/org/http4s/Message.scala

```diff
@@ -768,7 +768,7 @@ object Response extends KleisliSyntax {
   val notFound: Response[Pure] =
     Response(
       Status.NotFound,
-      entity = Entity.Strict(Chunk.array("Not found".getBytes(StandardCharsets.UTF_8))),
+      entity = Entity.Strict(ByteVector.view("Not found".getBytes(StandardCharsets.UTF_8))), // TODO
```

Comment:
I opened a PR to add literals e.g. `utf8Bytes"Not found"`.

- https://github.com/scodec/scodec-bits/pull/381